tezos-sig-gating-contracts/scripts/tests/gatednftminterSimpleMultisig.spec.ts

```diff
@@ -0,0 +1,358 @@
+      functionCallArgs.owner,
+      functionCallArgs.token_id
+    );
+    const payloadToSign: TezosTxAuthData = {
+      chainID: currentChainId,
+      userAddress: "tz1fon1Hp3eRff17X82Y3Hc2xyokz33MavFF",
+      nonce: 0,
+      blockExpiration: currentBlock + 10,
+      contractAddress: functionCallContract,
+      functionCallName: "%mint_gated",
+      functionCallArgs: functionCallArgsBytes,
+      signerPublicKey: nexeraSignerPublicKey,
+    };
+    const payloadHash = computePayloadHash(payloadToSign);
+    // Nexera signs Hash of payload
+    let signature = await nexeraSigner.sign(payloadHash);
+    // Execute mint-offchain entrypoint
+    const args: TezosTxCalldata = buildTxCallDataNoFunctionName(
+      payloadToSign,
+      signature.prefixSig
+    );
+    // CALL contract
+    const op = await cntr.methodsObject.mint_gated(args).send();
+    console.log(
+      `Waiting for Exec_gated_calldata on ${exampleGatedNFTMinter} to be confirmed...`
+    );
+    await op.confirmation(2);
+    console.log("tx confirmed: ", op.hash);
+
+    // VERIFY
+    const storage: any = await cntr.storage();
+    const admin = await storage.admin;
+    const ownerAsset0 = await storage.siggated_extension.ledger.get(0);
+    const ownerAsset1 = await storage.siggated_extension.ledger.get(1);
+    expect(deployerAddress === admin).to.be.true;
+    expect(ownerAsset0 === deployerAddress).to.be.true;
+    expect(ownerAsset1 === functionCallArgs.owner).to.be.true;
+    const userNonce = await storage.nonces.get(functionCallArgs.owner);
+    expect(userNonce.toNumber() === 1).to.be.true;
+  });
+
+  it(`Should add Frank as owner in multisig`, async () => {
+    const cntrSignerManager = await Tezos.contract.at(
+      exampleGatedNFTMinterCntrl ? exampleGatedNFTMinterCntrl : ""
+    );
+    const currentStorage: any = await cntrSignerManager.storage();
+    lastProposalId = await currentStorage.next_proposal_id;
+    const op = await cntrSignerManager.methodsObject
+      .createAddOwnerProposal(frank)
+      .send();
+    // const op = await cntrSignerManager.methodsObject.addOwner(frank).send();
+    await op.confirmation(2);
+    // Verify
+    const storageAfter: any = await cntrSignerManager.storage();
+    const prop = await storageAfter.proposals.get(lastProposalId);
+    expect(prop.action?.addOwner === frank).to.be.true;
+
+    //Tezos.setSignerProvider(await InMemorySigner.fromSecretKey(frank_pk));
+    const op_validate = await cntrSignerManager.methodsObject
+      .validateProposal([lastProposalId, true])
+      .send();
+    await op_validate.confirmation(2);
+    // Verify
+    const storage: any = await cntrSignerManager.storage();
+    const frankAuth = await storage.owners.get(frank);
+    expect(frankAuth).to.be.true;
+  });
+
+  it(`Should setThreshold to 2 in multisig`, async () => {
+    const cntrSignerManager = await Tezos.contract.at(
+      exampleGatedNFTMinterCntrl ? exampleGatedNFTMinterCntrl : ""
```

Comment:
the linter doesn't seem to work.. it should tell you to replace this with `exampleGatedNFTMinterCntrl ?? ""`
Also I think if exampleGatedNFTMinterCntrl is undefined you should probably throw an error because it should work with an empty address, right?

Reply:
agreed